constraint/pkg/client/client.go

```diff
@@ -767,6 +786,36 @@ func (c *Client) Dump(ctx context.Context) (string, error) {
 	return dumpBuilder.String(), nil
 }
 
+func (c *Client) GetDescriptionForStat(source instrumentation.Source, statName string) string {
+	if source.Type != instrumentation.EngineSourceType {
+		// only handle engine source for now
+		return instrumentation.UnknownDescription
+	}
+
+	// this is written in a general form
+	// but it only works for rego drivers for now.
+	for dName, d := range c.drivers {
+		if strings.EqualFold(dName, source.Value) {
+			if source.Value == instrumentation.RegoSource.Value {
+				// no other drivers implement the GetDescriptionForStat yet
```

Comment:
Add it to the Driver interface, updating the test driver as well.

There should be no need for type casting here.